### PR TITLE
fix bugs

### PR DESCRIPTION
**Pull request by lucasjesusone**

### CODE REVIEW

File: backend/src/controllers/checkoutController.js [hunk 1]
```diff
@@ -1,15 +1,12 @@
...
   Register(request, response) {
-    const span = tracer.scope().active()
     try{
       const { amount, type} = request.body;
       if (amount) {
-        const checkoutCreated = Checkout.create({ typerrrrr, amount });
+        const checkoutCreated = Checkout.create({ type, amount });
         response.status(200).json(checkoutCreated);
       } else{
         span.setTag('error.msg', "Valor de 'amount' chegou nulo");
```
Comment (jp-amis) [inline backend/src/controllers/checkoutController.js]: since you removed the tracer we need to get rid of this one too

**Review by vcwild — APPROVED** [review]
LGTM! Good job! :wink: 
You reached for the solution and also saved on resources by removing unused imports.

**Comment by FelipePassos** [comment]
was thinking about using DataDog on my system, thanks for making me give up

**Comment by doug-last** [comment]
common problem in the everyday life of any developer.

File: backend/src/controllers/checkoutController.js (hunk 1 above)
Comment (lucasjesusone) [reply backend/src/controllers/checkoutController.js]: LGTM too, I did it now. tks

**Comment by lucasjesusone** [comment]
image


works fine now

**Comment by willianvalerio** [comment]
HI @lucasjesusone, congrats for solving the challenge!

Since the goal of this repo is to provide a code with some bugs and provide a better error handling with datadog trace, I will close this PR.

**Pull request closed without merging**